src/launchpad/parsers/buffer_wrapper.py

```diff
@@ -29,13 +31,14 @@ def __exit__(
         exc_tb: types.TracebackType | None,
     ) -> None:
         """Exit debug group."""
-        logger.debug("=== End %s ===", self.name)
+        if self.enabled:
+            logger.debug("=== End %s ===", self.name)
 
 
 class BufferWrapper:
     """Wrapper for binary buffer parsing with cursor tracking and debugging."""
 
-    def __init__(self, buffer: bytes) -> None:
+    def __init__(self, buffer: bytes, debug: bool = False) -> None:
```

Comment:
Had to add this in because otherwise parsing takes forever locally. If you need it for debugging then can manually set.

Reply:
Fine with me, agree that the debug logging was out of hand without this